main/src/ui/manage_accounts/add_account_dialog.vala

```diff
@@ -339,6 +339,14 @@ public class AddAccountDialog : Gtk.Dialog {
             register_form_continue.visible = true;
             register_form_continue.grab_focus();
         } else if (form.fields.size > 0) {
+            if (form.title != null && form.title != "") {
```

Comment:
With this change, you display two titles: ours and the one chosen by the server. Please pick only one. The title goes into `register_title.label`.

Reply:
I know. I want to display the title from the server because it might be important for the registration. I also don't want it to control the entire content of the window. This is intended and I did like Gajim did as can be seen in the screenshot in https://github.com/dino/dino/issues/988.

Reply:
I understand both of your points. However, I think that showing both titles is not a satisfactory solution to the problem, since in most cases one title will be redundant.
While I don't have a good solution myself right now, I don't feel this change is an improvement to the status quo. My suggestion would be to display the server-instructions, but not the server-title for now. That way, the PR improves the current status without making anything worse.